examples/acados_matlab_octave/getting_started/new_minimal_example_ocp.m

```diff
@@ -0,0 +1,142 @@
+% ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+% LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+% CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+% SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+% INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+% CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+% ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+% POSSIBILITY OF SUCH DAMAGE.;
+
+import casadi.*
+
+
+%
+check_acados_requirements()
+
+%% solver settings
+N = 20; % number of discretization steps
+T = 1; % [s] prediction horizon length
+x0 = [0; pi; 0; 0]; % initial state
+
+sim_method = 'erk'; % erk, irk, irk_gnsf
+
+%% model dynamics
+model = get_pendulum_on_cart_AcadosModel();
+nx = length(model.x); % state size
+nu = length(model.u); % input size
+
+%% OCP formulation object
+ocp = AcadosOcp();
+ocp.model = model;
+
+%% cost in nonlinear least squares form
+W_x = diag([1e3, 1e3, 1e-2, 1e-2]);
+W_u = 1e-2;
+
+% initial cost term
+ocp.cost.cost_type_0 = 'NONLINEAR_LS';
+ocp.cost.W_0 = W_u;
+ocp.cost.yref_0 = zeros(nu, 1);
+ocp.model.cost_y_expr_0 = model.u;
+
+% path cost term
+ocp.cost.cost_type = 'NONLINEAR_LS';
+ocp.cost.W = blkdiag(W_x, W_u);
+ocp.cost.yref = zeros(nx+nu, 1);
+ocp.model.cost_y_expr = vertcat(model.x, model.u);
+
+% terminal cost term
+ocp.cost.cost_type_e = 'NONLINEAR_LS';
+ocp.model.cost_y_expr_e = model.x;
+ocp.cost.yref_e = zeros(nx, 1);
+ocp.cost.W_e = W_x;
+
+%% define constraints
+% only bound on u on initial stage and path
+ocp.model.con_h_expr = model.u;
+ocp.model.con_h_expr_0 = model.u;
+
+U_max = 80;
+ocp.constraints.lh = -U_max;
+ocp.constraints.lh_0 = -U_max;
+ocp.constraints.uh = U_max;
+ocp.constraints.uh_0 = U_max;
+ocp.constraints.x0 = x0;
+
+% define solver options
+ocp.dims.N = N;
+ocp.solver_options.tf = 1.0;
+ocp.solver_options.nlp_solver_type = 'SQP';
+ocp.solver_options.integrator_type = 'ERK';
+ocp.solver_options.qp_solver = 'PARTIAL_CONDENSING_HPIPM';
+ocp.solver_options.qp_solver_cond_N = 5;
+ocp.solver_options.hessian_approx = 'GAUSS_NEWTON';
+ocp.solver_options.ext_fun_compile_flags = '-O2';
+ocp.solver_options.globalization = 'MERIT_BACKTRACKING';
+% ocp.solver_options.qp_solver_iter_max = 100
+
+% create solver
+ocp_solver = AcadosOcpSolver(ocp);
+
+% solver initial guess
+x_traj_init = zeros(nx, N+1);
+u_traj_init = zeros(nu, N);
+
+%% call ocp solver
+% update initial state
+ocp_solver.set('constr_x0', x0);
+
+% set trajectory initialization
+ocp_solver.set('init_x', x_traj_init); % states
+ocp_solver.set('init_u', u_traj_init); % inputs
+ocp_solver.set('init_pi', zeros(nx, N)); % multipliers for dynamics equality constraints
+
+% change values for specific shooting node using:
+%   ocp_solver.set('field', value, optional: stage_index)
+ocp_solver.set('constr_lbx', x0, 0)
```

Comment:
should this maybe be a comment? this will overwrite the bounds set with  `ocp_solver.set('constr_x0', x0);`  a few lines above